rust/agama-cli/doc/backend-for-testing.md

```diff
@@ -1,21 +1,21 @@
-# How to set up a backend for testing this CLI frontend
+# How to set up a backend for testing the CLI frontend
 
-I needed a testing instance of the D-Installer backend so that this
-command-line frontend has something to talk to.
+I needed a testing instance of the Agama backend so that the
+Rust command-line frontend has something to talk to.
 
 ## Summary
 
 1. Take the container used for continuous integration (CI) testing of the
    backend
-2. Give it a git checkout of the backend
+2. Give it a git checkout of this repo
 3. Install the backend within the container
 4. Copy the frontend binary into the container
 
 ## Considered Alternatives
 
 My first plan had a different finale, 4. Make the D-Bus service visible
 ouside the container, but I hit an issue with D-Bus authentication, hopefully
-solvable.
+solvable. (Update: `xdg-dbus-proxy` seems to work, ask mvidner about it)
 
 Josef wanted to test against a different container ([d-installer-backend][]) but that one was a
```

Comment:
Mentioning Josef is... just giving credit.
But the thing he tried, I want to take a closer look: we have ~~two~~ 3 similar containers
- https://build.opensuse.org/package/show/YaST:Head:Containers/agama-testing "Specific Agama container used for CI testing (descr: Highly unstable)" ... which I use here
- https://build.opensuse.org/package/show/YaST:Head:Containers/agama-service (no description)
- https://build.opensuse.org/package/show/YaST:Head:Containers/agama-all (no description)


If we add a summary + description to these, it will be more apparent to visitors AND team members why we have all these similar yet different packagings. @imobachgs ?